docs/4.1/admin-guide.md

```diff
@@ -2081,16 +2081,10 @@ teleport:
   storage:
       # The region setting sets the default AWS region for all AWS services 
       # Teleport may consume (DynamoDB, S3)
-      region: us-west-1
+      region:  us-east-1 
 
-      # Path to S3 bucket to store the recorded sessions in. The optional 'region'
-      # parameter allows to override the region setting above, keeping S3 recordings
-      # in a different region:
-      audit_sessions_uri: s3://example.com/path/to/bucket?region=us-east-1
-
-      # Authentication settings are optional (see below)
-      access_key: BKZA3H2LOKJ1QJ3YF21A
```

Comment:
instead of removed `access_key` please add a note about the way teleport assumes the credentials (using provider chains assuming IAM mounted file or standard .awd/credentials in the home folder)

Reply:
@benarent ping